app/src/main/java/org/wikipedia/dataclient/mwapi/CreateAccountResponse.kt

```diff
@@ -0,0 +1,32 @@
+package org.wikipedia.dataclient.mwapi
+
+class CreateAccountResponse : MwResponse() {
+
+    private val createaccount: Result? = null
+
+    fun status(): String? {
+        return createaccount!!.status
+    }
+
+    fun user(): String? {
+        return createaccount!!.username
+    }
+
+    fun message(): String? {
+        return createaccount!!.message
+    }
```

Comment:
Would it be possible to change the format to `val status get() = createaccount?.status`. And some null-related checks maybe need to revisit where they are currently on.